Device: cuda
CUDA Version: 12.1
cuDNN Version: 8902
PyTorch Version: 2.3.0

Model used: PACSCNN_4
Initial dataset creation
Initial dataset creation
Initial loss: 0.125986710190773
<__main__.Policy object at 0x148c74a295b0>
0.125986710190773
Average loss over 5 updates: 0.2201
Round 1 took 1.3637218475341797s, results:
{'t': 0, 'current_accuracy': 0.9296875, 'current_loss': 0.21914756670594215, 'train_loss': 0.22009294033050536, 'decision': 1, 'drift_rate': 0.0, 'target_domains': ['photo']}
Traceback (most recent call last):
  File "/home/apiasecz/programming/concept_drift_optimal_adaptation_FL/src/execute/evaluate_policy.py", line 899, in <module>
    main()
  File "/home/apiasecz/programming/concept_drift_optimal_adaptation_FL/src/execute/evaluate_policy.py", line 879, in main
    evaluate_policy_under_drift(
  File "/home/apiasecz/programming/concept_drift_optimal_adaptation_FL/src/execute/evaluate_policy.py", line 663, in evaluate_policy_under_drift
    drift_rate, target_domains = drift_scheduler.get_drift_params(t)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/apiasecz/programming/concept_drift_optimal_adaptation_FL/src/execute/evaluate_policy.py", line 452, in get_drift_params
    self.limits_interval = (self.burst_interval[0], self.burst_interval[1])
                            ~~~~~~~~~~~~~~~~~~~^^^
TypeError: 'int' object is not subscriptable
